src/PlatformBar.js

```diff
@@ -11,14 +12,24 @@ export default class ConsoleBar extends Component {
     let checked = [];
     const checkboxes = document.querySelectorAll('.platformCheckBox:checked')
     checkboxes.forEach((elem) => {
-        checked.push(elem.value)
+      checked.push(elem.value)
     })
-    this.setState({platformsToFilter: checked});
-    this.props.updateState({platformsToFilter: this.state.platformsToFilter})
-    }
+    this.setState({ platformsToFilter: checked });
+    this.props.updateState({ platformsToFilter: this.state.platformsToFilter })
```

Comment:
Hm, little confused by this `updateState` method -- it seems to be setting the same value in state as `this.setState` -- what's different about these two lines of code? Can we refactor it to be a bit more readable or are we doing some extra steps that we might be able to avoid?